backend/.nvmrc

```diff
@@ -0,0 +1 @@
+v18.7.0
```

Comment:
I know this modul will work with V20.0.0 too.
We should plan an upgrade for all moduls as soon as possible, right?

Reply:
Yes right, but in another pr.